### PR TITLE
Muiv5 upgrade

### PR DESCRIPTION
**Pull request by aachavarria**
Confirm Dropdown
RenderTreeNodes
PathNavigatorTree: Fixing issues related to onLabel and onIcon click;
Fixing issues related to background color paper on inputs and dialog;

### CODE REVIEW

File: ui/app/src/components/Controls/ConfirmDropdown.tsx [hunk 1]
```diff
@@ -104,19 +98,28 @@ export default function ConfirmDropdown(props: ConfirmDropdownProps) {
     onCancel?.();
   };
 
+  const iconButton = useMemo(
```
Comment (rart) [inline ui/app/src/components/Controls/ConfirmDropdown.tsx:102]:
```suggestion
    // Not memoizing causes the menu to get misplaced upon opening.
    const iconButton = useMemo(
```

File: ui/app/src/components/Dialogs/DialogBody.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ const styles = makeStyles((theme: Theme) =>
       display: 'flex',
       flex: '1 1 auto',
       flexDirection: 'column',
-      padding: theme.spacing(2),
+      padding: `${theme.spacing(2)} !important`,
```
Comment (rart) [inline ui/app/src/components/Dialogs/DialogBody.tsx:32]: This needs to change to the emotion-based API

File: ui/app/src/components/FolderBrowserTreeView/RenderTreeNode.tsx [hunk 3]
```diff
@@ -44,16 +47,15 @@ export function RenderTreeNode(props: RenderTreeNodeProps) {
...
         root: classes.treeItemRoot,
         content: classes.treeItemContent,
         selected: classes.treeItemSelected,
         label: clsx(classes.treeItemLabel, props.classes?.treeItemLabel)
       }}
-      // TODO: Handle label/icon click
-      // onIconClick={(e) => onIconClick?.(e, node)}
-      // onLabelClick={(e) => onLabelClick?.(e, node)}
+      expandIcon={<ExpandMoreIcon onClick={(e) => onIconClick?.(e, node)} />}
+      collapseIcon={<ChevronRightIcon onClick={(e) => onIconClick?.(e, node)} />}
```
Comment (rart) [inline ui/app/src/components/FolderBrowserTreeView/RenderTreeNode.tsx]: add role="button" to icons

File: ui/app/src/components/FolderBrowserTreeView/RenderTreeNode.tsx [hunk 4]
```diff
@@ -44,16 +47,15 @@ export function RenderTreeNode(props: RenderTreeNodeProps) {
     <TreeItem
       key={node.id}
       nodeId={node.id}
-      label={node.name}
+      label={<div onClick={(e) => onLabelClick?.(e, node)}>{node.name}</div>}
```
Comment (rart) [inline ui/app/src/components/FolderBrowserTreeView/RenderTreeNode.tsx]: add role=button

File: ui/app/src/components/PathNavigatorTree/PathNavigatorTreeItem.tsx [hunk 5]
```diff
@@ -84,7 +90,8 @@ const useStyles = makeStyles((theme) =>
       width: '100%',
       display: 'flex',
       alignItems: 'center',
-      height: '26px',
+      // height: '26px',
```
Comment (rart) [inline ui/app/src/components/PathNavigatorTree/PathNavigatorTreeItem.tsx]: x

File: ui/app/src/components/PathNavigatorTree/PathNavigatorTreeUI.tsx [hunk 6]
```diff
@@ -130,13 +128,7 @@ export default function PathNavigatorTreeUI(props: PathNavigatorTreeUIProps) {
...
       />
       <AccordionDetails className={clsx(classes.accordionDetails, props.classes?.body)}>
-        <TreeView
-          className={classes.root}
-          expanded={expandedNodes}
-          defaultExpandIcon={<ArrowRightRoundedIcon />}
-          defaultCollapseIcon={<ArrowDropDownRoundedIcon />}
-          disableSelection={true}
-        >
+        <TreeView className={classes.root} expanded={expandedNodes} disableSelection={true}>
```
Comment (rart) [inline ui/app/src/components/PathNavigatorTree/PathNavigatorTreeUI.tsx]:
```suggestion
        <TreeView className={classes.root} expanded={expandedNodes} disableSelection>
```

File: ui/app/src/components/SiteConfigurationManagement/SiteConfigurationManagement.tsx [hunk 7]
```diff
@@ -585,7 +585,7 @@ export default function SiteConfigurationManagement(props: SiteConfigurationMana
                     <SecondaryButton disabled={encrypting} onClick={onEncryptClick} loading={encrypting}>
                       {formatMessage(translations.encryptMarked)}
                     </SecondaryButton>
-                    <Button size="small" onClick={onEncryptHelpClick}>
+                    <Button size="small" color="inherit" onClick={onEncryptHelpClick}>
```
Comment (rart) [inline ui/app/src/components/SiteConfigurationManagement/SiteConfigurationManagement.tsx]: What's the inherit for?

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/components/Dialogs/DialogBody.tsx (hunk 2 above)
Comment (aachavarria) [reply ui/app/src/components/Dialogs/DialogBody.tsx:32]: I will work this after merge, with another pending changes

File: ui/app/src/components/PathNavigatorTree/PathNavigatorTreeItem.tsx [hunk 8]
```diff
@@ -245,17 +257,21 @@ export default function PathNavigatorTreeItem(props: PathNavigatorTreeItemProps)
...
+          collapseIcon={<ArrowDropDownRoundedIcon onClick={() => onIconClick(node.id)} />}
           label={
             <>
-              <section className={classes.itemDisplaySection} onMouseOver={onMouseOver} onMouseLeave={onMouseLeave}>
+              <section
+                onClick={(event) => onLabelClick(event, node.id)}
+                className={classes.itemDisplaySection}
+                onMouseOver={onMouseOver}
+                onMouseLeave={onMouseLeave}
+              >
```
Comment (rart) [inline ui/app/src/components/PathNavigatorTree/PathNavigatorTreeItem.tsx:269]:
```suggestion
              <section
                role="button"
                onClick={(event) => onLabelClick(event, node.id)}
                className={classes.itemDisplaySection}
                onMouseOver={onMouseOver}
                onMouseLeave={onMouseLeave}
              >
```

File: ui/app/src/components/PathNavigatorTree/PathNavigatorTreeItem.tsx [hunk 9]
```diff
@@ -245,17 +257,21 @@ export default function PathNavigatorTreeItem(props: PathNavigatorTreeItemProps)
         <TreeItem
           key={node.id}
           nodeId={node.id}
-          // TODO: replace onLabelClick & onIconClick
-          // onLabelClick={(event) => onLabelClick(event, node.id)}
-          // onIconClick={() => onIconClick(node.id)}
+          expandIcon={<ArrowRightRoundedIcon onClick={() => onIconClick(node.id)} />}
+          collapseIcon={<ArrowDropDownRoundedIcon onClick={() => onIconClick(node.id)} />}
```
Comment (rart) [inline ui/app/src/components/PathNavigatorTree/PathNavigatorTreeItem.tsx:261]:
```suggestion
          expandIcon={<ArrowRightRoundedIcon role="button" aria-label={formatMessage(translations.expand)} aria-hidden="false" onClick={() => onIconClick(node.id)} />}
          collapseIcon={<ArrowDropDownRoundedIcon role="button" aria-label={formatMessage(translations.collapse)} aria-hidden="false" onClick={() => onIconClick(node.id)} />}
```

File: ui/app/src/components/FolderBrowserTreeView/RenderTreeNode.tsx [hunk 10]
```diff
@@ -44,16 +47,19 @@ export function RenderTreeNode(props: RenderTreeNodeProps) {
...
         root: classes.treeItemRoot,
         content: classes.treeItemContent,
         selected: classes.treeItemSelected,
         label: clsx(classes.treeItemLabel, props.classes?.treeItemLabel)
       }}
-      // TODO: Handle label/icon click
-      // onIconClick={(e) => onIconClick?.(e, node)}
-      // onLabelClick={(e) => onLabelClick?.(e, node)}
+      expandIcon={<ExpandMoreIcon role="button" onClick={(e) => onIconClick?.(e, node)} />}
+      collapseIcon={<ChevronRightIcon role="button" onClick={(e) => onIconClick?.(e, node)} />}
```
Comment (rart) [inline ui/app/src/components/FolderBrowserTreeView/RenderTreeNode.tsx:62]:
```suggestion
      expandIcon={<ExpandMoreIcon role="button" aria-label={formatMessage(translations.expand)} aria-hidden="false" onClick={(e) => onIconClick?.(e, node)} />}
      collapseIcon={<ChevronRightIcon role="button" aria-label={formatMessage(translations.collapse)} aria-hidden="false" onClick={(e) => onIconClick?.(e, node)} />}
```

File: ui/app/src/components/DeleteContentTypeDialog/DeleteContentTypeDialogContainer.tsx [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+        });
+        const response = e.response?.response ?? e.response;
+        dispatch(
+          showSystemNotification({
+            message: response?.message ?? formatMessage(messages.deleteFailed),
+            options: { variant: 'error' }
+          })
+        );
+      }
+    );
```
Comment (rart) [inline ui/app/src/components/DeleteContentTypeDialog/DeleteContentTypeDialogContainer.tsx:64]: This subscribe signature is deprecated (as the editor should have shown you). We must use:
```suggestion
    deleteContentType(site, contentType.id).subscribe({
      next() {
        onSubmittingAndOrPendingChange({
          isSubmitting: false
        });
        dispatch(showSystemNotification({ message: formatMessage(messages.deleteComplete) }));
        onComplete?.();
      },
      error(e) {
        onSubmittingAndOrPendingChange({
          isSubmitting: false
        });
        const response = e.response?.response ?? e.response;
        dispatch(
          showSystemNotification({
            message: response?.message ?? formatMessage(messages.deleteFailed),
            options: { variant: 'error' }
          })
        );
      }
    });
```

**Review by rart — CHANGES_REQUESTED** [review]

**Comment by rart** [comment]
Please fix comments on the following PR

**Pull request merged**